cmd/transcriber/call/transcriber.go

```diff
@@ -31,6 +34,10 @@ type Transcriber struct {
 	liveTracksWg sync.WaitGroup
 	trackCtxs    chan trackContext
 	startTime    atomic.Pointer[time.Time]
+
+	transcriberQueueCh chan captionPackage
+	transcriberWg      sync.WaitGroup
+	transcriberDoneCh  chan struct{}
```

Comment:
Could we rename them slightly to be more explicit as these refer to the transcriber pool for live captions. Or add a comment.

Reply:
Changed to captionsPool, thanks.